notebooks/00_quick_start/sar_movielens.ipynb

```diff
@@ -15,14 +15,14 @@
    "source": [
```

Comment:
<p>Minor comment, I think `PythonRankingEvaluation` is the old name that we were giving the the python ranking metrics, now there is no class, just functions</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>gotcha, changed</p>